profile/README.md

```diff
@@ -0,0 +1,66 @@
+# Lombiq Technologies 
+
+
+
+[Lombiq Technologies](https://lombiq.com) is a software, training, and services company focusing on web development with open Microsoft technologies. 
+Our main area of expertise is the [Orchard Core content management system](https://www.orchardcore.net/). We provide custom development, training, hosting, and consulting services and we are behind the first (and only) Orchard CMS SaaS, [DotNest](https://dotnest.com).
+
+
+## Our services:
+
+### Software development: 
+Our team's passion is developing software with care for clients who need help with their most extraordinary projects in the following frameworks:
+
+- Orchard Core
+- Orchard CMS
+- ASP.NET Core
+- ASP.NET
+
+### Consulting:
+We offer Orchard consulting in both Orchard editions, Orchard Core and Orchard CMS, including website and application development, and code review.
+
+### Training:
+Our team of dedicated experts is here to help groups and individuals who want to learn about Orchard or simply broaden their coding knowledge. Our [Orchard Dojo](https://orcharddojo.net/) website contains several training sessions to help beginners become experts.
+
+![Orchard Dojo](https://user-images.githubusercontent.com/68704777/174299406-a7ea1222-a1a8-457e-b68f-e3570bf2d5b6.png)
+
+
+**We also contribute a vast amount of code, documentation, and learning resources to the open-source ecosystem. These materials are also available for our partners to use even after we have delivered a successful project.**
+
+
+## Products:
+
+* [**Hastlayer**](https://hastlayer.com/) turns performance-critical sections of .NET programs into computer chips (FPGA). You can still update your software, but it will be a whole lot faster and more power-efficient.
+  
+![Hastlayer_logo](https://user-images.githubusercontent.com/68704777/174298010-59da9056-74f0-48a5-a6f3-ff3f93a2b346.png)
+
+* [**DotNest**](https://dotnest.com/) We've developed DotNest as a software-as-a-service for Orchard CMS, enabling users to create their websites using Orchard in a flash. Today it runs more than 6000 websites.
+
+ ![DotNest logo medium](https://user-images.githubusercontent.com/68704777/174301139-3c4ab813-8fe8-4ca1-a428-d9bf4cd98bdb.png)
+
+
+## Our team
+
+Lombiq's team consists of highly skilled experts from diverse backgrounds:
+ 
+ ![lombiq team 2021 summer](https://user-images.githubusercontent.com/68704777/174300662-93d75ff3-5c7d-48c4-9236-775634402287.jpg)
```

Comment:
You should be able to reference this image directly with the path "lombiq-team-2021-summer_50.jpg". Also, rather name the file just "lombiq-team-2021-summer.jpg"